packages/db/src/index.ts

```diff
@@ -18,3 +37,32 @@ export const dropAllTables = async ({
     tableNames.map((tableName) => knex.schema.dropTable(tableName))
   );
 };
+
+export const getConnectionFromEnvironment = async (
+  env: NodeJS.ProcessEnv
+): Promise<Knex> => {
+  let connectionConfig = {};
+  if (env?.databaseCredentialSecretId === undefined) {
+    connectionConfig = await getEnvConnectionConfig(env);
+  } else {
+    connectionConfig = await getSecretConnectionConfig(env.databaseCredentialSecretId);
+  }
+
+  let knexConfig = {
+    client: 'pg',
+    connection: connectionConfig,
+    debug: env?.KNEX_DEBUG === 'true',
+    asyncStackTraces: env?.KNEX_ASYNC_STACK_TRACES === 'true',
+    acquireConnectionTimeout: 60000,
```

Comment:
I noticed that in one of the Lambdas this value was set to 120000. Does that matter?

Reply:
It may.  I'll comment in standup - also this should still be a draft (it's not set for review), although feel free to comment.

Reply:
Or not.    So - I'm not convinced this timeout is actually isolating us from DB spin-up timeouts.     I've upped the db spin-down time to eliminate that issue for now and am going to dig into what's going on there on a test cluster - it's probable I'm just not fully grokking what that setting does in the underlying  pg driver.       60 seconds *should* be more than enough for a RDS spin-up.